WalletWasabi.Fluent/Views/Dialogs/Announcement/ZkSnacksCoordinatorAnnouncementView.axaml

```diff
@@ -8,11 +8,25 @@
              x:CompileBindings="True"
              x:Class="WalletWasabi.Fluent.Views.Dialogs.Announcement.ZkSnacksCoordinatorAnnouncementView">
   <ContentArea Title="{Binding Title}"
-               Caption="zkSNACKs coordiantor stopped."
                EnableCancel="{Binding EnableCancel}"
                EnableBack="{Binding EnableBack}"
                EnableNext="True" NextContent="Continue">
+    <DockPanel>
 
+      <Button Margin="0 20 0 0" Command="{Binding OpenSettingsCommand}" Content="Open Settings" Classes="h7 plain activeHyperLink" HorizontalAlignment="Center" DockPanel.Dock="Bottom" />
+
+      <TextBlock DockPanel.Dock="Bottom"
+                 MaxWidth="400"
+                 Margin="25 0"
+                 TextWrapping="Wrap"
+                 TextAlignment="Center"
+                 Text="The default coinjoin coordination service provided by zkSNACKs is discontinued as of June 1st 2024. To continue using the coinjoin feature, add a new coordinator to your application’s Bitcoin settings." />
```

Comment:
nit feel free to ignore 
```suggestion
                 Text="The default coinjoin coordination service provided by zkSNACKs is discontinued as of June 1st 2024. To continue using the coinjoin feature, enter a new coordinator in the Settings." />
```

Reply:
_in the Settings_ or _in the Bitcoin Settings_

Reply:
Something is odd, I pressed commit but nothing happened, and now it thinks that it is committed.

![image](https://github.com/WalletWasabi/WalletWasabi/assets/16364053/62719aa8-86fb-4d75-bd05-9bb786975654)